### ISSUE TITLE
Are you guys real?

### ISSUE THREAD

**Issue by SilverEzhik** [nodeschool/baynode#1]

**Comment by rosskukulinski** [comment]
Yep - We're real. I'll make a note to complete the process of setting up this repo later this week.

[This](http://www.meetup.com/BayNode/) is us.
